core/src/main/java/com/valtech/aemsaas/core/models/search/PaginationQuery.java

```diff
@@ -0,0 +1,41 @@
+package com.valtech.aemsaas.core.models.search;
+
+import org.apache.commons.lang3.StringUtils;
+
+public final class PaginationQuery implements FulltextSearchGetQuery {
+
+  static final String START = "start";
```

Comment:
those params need to be documented as they will need to be whitelisted in the dispatcher configs

Reply:
The params are already on confluence. I am using https://decks.valtech.swiss/display/ICSAAS/Search+APIv3 as reference